transformers4rec/torch/model/prediction_task.py

```diff
@@ -36,7 +36,7 @@ def build(self, input_size) -> SequentialBlock:
         return SequentialBlock(
             torch.nn.Linear(input_size[-1], 1, bias=False),
             torch.nn.Sigmoid(),
-            LambdaModule(lambda x: x.view(-1)),
+            LambdaModule(lambda x: torch.squeeze(x, -1)),
```

Comment:
In the case of sequential binary classification, we'll need to keep the second dimension (i.e. sequence length)